spec/requests/api/v1/users_spec.rb

```diff
@@ -12,11 +12,16 @@
     before :each do
       # Heads up, for some reason the stored CIP IDs don't match
       cip = create(:core_induction_programme, name: "Teach First")
+      cohort = create(:cohort)
```

Comment:
You don't need this, the school_cohort factory will create it's own cohort if you don't provide one